main.go

```diff
@@ -14,50 +14,21 @@ import (
 	"github.com/antonlindstrom/mesos_stats"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/log"
+	"github.com/pvdissel/mesos_metrics"
 )
 
 const concurrentFetch = 100
 
 // Commandline flags.
 var (
-	addr           = flag.String("web.listen-address", ":9105", "Address to listen on for web interface and telemetry")
-	autoDiscover   = flag.Bool("exporter.discovery", false, "Discover all Mesos slaves")
-	localURL       = flag.String("exporter.local-url", "http://127.0.0.1:5051", "URL to the local Mesos slave")
-	masterURL      = flag.String("exporter.discovery.master-url", "http://mesos-master.example.com:5050", "Mesos master URL")
-	metricsPath    = flag.String("web.telemetry-path", "/metrics", "Path under which to expose metrics")
+	addr = flag.String("web.listen-address", ":9105", "Address to listen on for web interface and telemetry")
+	autoDiscover = flag.Bool("exporter.discovery", false, "Discover all Mesos slaves")
+	localURL = flag.String("exporter.local-url", "http://127.0.0.1:5051", "URL to the local Mesos slave")
+	masterURL = flag.String("exporter.discovery.master-url", "http://mesos-master.example.com:5050", "Mesos master URL")
```

Comment:
It works just as it did before.
If you use the exporter.discovery option with exporter.discovery.master-url, it will ask the elected master for its slaves and will scrape that master and its slaves.
If you provide only the exporter.local-url it will scrape only that specific node as slave.

If you run an exporter for each master, that will work fine, as it will automatically go to the elected master.
So yes, this will allow running one of these beside each master :smile: